### ISSUE TITLE
Can't extract correct confidence, label and num of faces for MobileNet SSD v2 (Faces)

### ISSUE THREAD

**Issue by michisaak** [tensorflow/models#7375]
Hi all,
I am working on a face detection project using a neural network and I am trying to extract the parameters (coordinates, confidence, label, number of faces). For the face boxes I have managed to get the coordinates using the following structure :
        for (int i = 0; i < 50; i++) {
              int xMin = (int) width * detectionResult[offset + i * 6 + 1];
              int yMin = (int) height * detectionResult[offset + i * 6 + 0];
              int xMax = (int) width * detectionResult[offset + i * 6 + 3];
              int yMax = (int) height * detectionResult[offset + i * 6 + 2];
        }
Now for the labels and the  confidence I am using : 
              float conf = (float) detectionResult[offset + i * 6 + 4];
              const int label = (int) detectionResult[offset + i * 6 + 5];
However this does not seem to give the right value. I have also tried via versa but still no luck. 
So maybe I am doing something wrong with the conversions? Are there any tricks that could help me finding those? 
I am using MobileNet SSD v2 (Faces) model. I use 50 in the loop because I cannot obtain the correct number of faces as well.

**Comment by tensorflowbutler** [comment]
Thank you for your post. We noticed you have not filled out the following field in the issue template. Could you update them if they are relevant in your case, or leave them as N/A? Thanks.
What is the top-level directory of the model you are using
Have I written custom code
OS Platform and Distribution
TensorFlow installed from
TensorFlow version
Bazel version
CUDA/cuDNN version
GPU model and memory
Exact command to reproduce

**Comment by PythonImageDeveloper** [comment]
Hi @michisaak 
Is is possible to explain how to run this model?
And please tell me your tensorflow version and GPU?

**Comment by ravikyram** [comment]
Automatically closing due to lack of recent activity. Please update the issue when new information becomes available, and we will reopen the issue. Thanks!
